app/models/like.rb

```diff
@@ -1,3 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :user, class_name: 'User'
```

Comment:
- [ ] A like belongs to the author, kindly consider using author instead of user e.g `belongs_to :author, class_name: 'User'
`